### ISSUE TITLE
First line of content is hidden under the textarea titles

### ISSUE THREAD

**Issue by badmansan** [WinMerge/winmerge#379]
Good day!
WinMerge 2.16.6.0, windows 7 x64.

How to reproduce.
1) Start program
2) Open new documents (File -> New), press Enter
3) See that first line & cursor hidden:
![img](https://i.imgur.com/68vM7yx.png)

**Comment by sdottaka** [comment]
You see it that way, probably because you have set the Diff Context to 0 or something else.

What if I set Diff Context to All Lines like this:

![image](https://user-images.githubusercontent.com/98126/88285771-a04ddf80-cd2a-11ea-9687-e1c407158b5c.png)

**Comment by badmansan** [comment]
> You see it that way, probably because you have set the Diff Context to 0 or something else.
Yes, that's it. Choosing `All Lines` solves the problem. But there is another problem.
Let's set those option to 3 lines, for example. When we only past content, the first line is hidden in both sides. If we want compare two one-line strings, we can't see original text, because it completely hide in first line.

**Comment by sdottaka** [comment]
The workaround is to enable the Automatic Rescan option as follows.

I don't have a good idea to deal with this issue right now.

![image](https://user-images.githubusercontent.com/98126/88290720-27eb1c80-cd32-11ea-86c0-825989944391.png)

**Comment by badmansan** [comment]
Thank you, `Automatic Rescan` make my life more easy.
I will leave this topic open, you can decide if you want to close this issue or try to fix in future.
